client/src/components/NavBar/NavBar.tsx

```diff
@@ -5,37 +5,59 @@ import logo from '../../Images/logo.png';
 import CircularProgress from '@material-ui/core/CircularProgress';
 import { useAuth } from '../../context/useAuthContext';
 import { useSocket } from '../../context/useSocketContext';
-import { useHistory } from 'react-router-dom';
-import { useEffect } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { useEffect, useRef } from 'react';
 import LoggedInBar from './AuthBars/LoggedInBar';
 import LoggedOutBar from './AuthBars/LoggedOutBar';
 import CssBaseline from '@material-ui/core/CssBaseline';
+import HomePageBar from './AuthBars/HomePageBar';
 
 const NavBar = (): JSX.Element => {
   const classes = useStyles();
 
   const { loggedInUser } = useAuth();
   const { initSocket } = useSocket();
+  const outterNavStyle = useRef(classes.appbarHomePage);
+  const innerNavStyle = useRef(classes.appbarHomePage);
 
   const history = useHistory();
+  const location = useLocation();
 
   useEffect(() => {
+    console.log('**************');
     initSocket();
   }, [initSocket]);
 
+  useEffect(() => {
+    console.log('**************');
```

Comment:
Let remove this one too

Reply:
resolved by commit 8cf73207227fbc2ac8aa4489e71b28120227c004